contrib/src/main/java/com/datatorrent/contrib/hbase/HBaseWindowStore.java

```diff
@@ -132,8 +140,11 @@ public void beginTransaction() {
 
   @Override
   public void commitTransaction() {
-    // HBase does support transactions so this method left empty
-
+    try {
+      flushTables();
```

Comment:
Also these other functions have a comment "HBase does support transactions so this method left empty" . Seems to be a typo with "not" missing. Can we fix these comemnts?